app/controllers/rent_controller.rb

```diff
@@ -1,6 +1,13 @@
 class RentController < ApplicationController
   include DeviseTokenAuth::Concerns::SetUserByToken
   # include Wor::Paginate
+  before_action :authenticate_user!
+  def index
+    # user = User.find_by!(email: request.headers['uid'].to_s)
+    # render_paginated Rent.where('user_id=' + user.id.to_s)
+
+    render_paginated Rent.where('user_id=' + current_user.id.to_s)
```

Comment:
NEVER use concatenation in your SQL queries, because you are opening the door to SQL Injection. Here is some reading about it https://rails-sqli.org/
In this case you are safe because you are concatenating a value you know, but if you do this with a value submitted by the user you will be introducing a big security bug.
The "right way" of doing this is
```suggestion
    render_paginated Rent.where(user: current_user)
```

Another way is 

```suggestion
    render_paginated current_user.rents
```